test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling.lua

```diff
@@ -874,6 +906,28 @@ function m.buttonSubscription(pAppId)
     end)
 end
 
+--[[ @getInteriorVehicleData: adding subscription for interior vehicle data
+--! @parameters:
+--! pAppId - application number (1, 2, etc.)
+--! isIVDCashed - false (default), true - if it's expected RC.GetInteriorVehicleData request on HMI
```

Comment:
@GetmanetsIrina The parameter's description is incorrect
 ```suggestion
--! isIVDCashed - false (default), true - if InteriorVehicleData is cached on SDL, it is no request performed to HMI
```